### PR TITLE
Sample manifests

### PR DESCRIPTION
**Pull request by tsibley**
Work to allow us to parse, diff, upload, and process sample manifests into the data warehouse. Commit messages contain all the details.

The parsed manifests are archived in and uploaded from https://github.com/seattleflu/specimen-manifests.

There are some manual steps needed after deployment of this branch:

- [ ] Remove existing samples/targets (and thus presence/absence) so the reprocessing of received data has a clean slate for the new ETL cooperation. `truncate warehouse.sample, warehouse.target, warehouse.presence_absence;`
- [ ] Run new enrollment ETL to reprocess all enrollments with new sample handling.
- [ ] Upload new manifest records with `id3c manifest upload`.
- [ ] Run manifest ETL to create/update samples.
- [ ] Run new presence/absence ETL to reprocess all enrollments with new sample handling.

### CODE REVIEW

**Comment by tsibley** [comment]
[Add sample barcode to shipping view](https://trello.com/c/GcLoXAmm/80-add-sample-barcode-to-shipping-view)

File: lib/seattleflu/db/cli/command/etl/enrollments.py [hunk 1]
```diff
@@ -103,6 +115,57 @@ def etl_enrollments(*, action: str):
...
+                    # XXX TODO: Stop hardcoding this and handle other types.
+                    # ScannedSelfSwab and ManualSelfSwabbed are kit barcodes,
+                    # not collection barcodes.  TestStrip is an identifier
+                    # UUID, not barcode.
+                    #   - trs, 17 May 2019
+                    if sample["type"] != "ClinicSwab":
+                        LOG.warning(f"Skipping collected sample with unknown type {sample['type']}")
+                        continue
+
+                    LOG.debug(f"Looking up collected sample code «barcode»")
```
Comment (kairstenfay) [inline lib/seattleflu/db/cli/command/etl/enrollments.py]: it looks like you forgot to wrap barcode here in curly brackets

Comment (tsibley) [reply lib/seattleflu/db/cli/command/etl/enrollments.py]: So I did! I pushed a fixup commit addressing that.

**Comment by kairstenfay** [comment]
Good work, Thomas! I'd love to sit down and talk about how/why you organized files the way you did once we have some free time (e.g. what merits a private function? a function in a utils file?).

**Comment by tsibley** [comment]
Thanks for giving this a look, @kairstenfay! I'd be happy to sit down and chat later this week or next.

**Pull request merged**